schematic/schemas/data_model_jsonld.py

```diff
@@ -0,0 +1,284 @@
+from functools import wraps
+from typing import Any, Dict, Optional, Text, List
+import networkx as nx
+
+from schematic.schemas.data_model_graph import DataModelGraphExplorer
+from schematic.schemas.data_model_relationships import DataModelRelationships
+from schematic.utils.schema_utils import get_label_from_display_name, convert_bool_to_str
+
+
+class DataModelJsonLD(object):
+    '''
+    #Interface to JSONLD_object
+    '''
+
+    def __init__(self, Graph: nx.MultiDiGraph, output_path:str = ''):
+        # Setup
+        self.graph = Graph
+        self.dmr = DataModelRelationships()
+        self.rel_dict = self.dmr.relationships_dictionary
+        self.DME = DataModelGraphExplorer(self.graph)
+        self.output_path = output_path
+
+    
+    def base_jsonld_template(self) -> dict:
+        """Base starter JSONLD template, to be filled out with model. For entire file.
+        Returns:
+            base_template, dict: base JSONLD template
+        TODO: when done adding contexts fill out this section here.
+        """
+        base_template = {
+                        "@context": {
+                            "bts": "http://schema.biothings.io/",
+                            "rdf": "http://www.w3.org/1999/02/22-rdf-syntax-ns#",
+                            "rdfs": "http://www.w3.org/2000/01/rdf-schema#",
+                            "schema": "http://schema.org/",
+                            "xsd": "http://www.w3.org/2001/XMLSchema#",
+                            },
+                         "@graph": [],
+                         "@id": "http://schema.biothings.io/#0.1",
+                        }
+        return base_template
+
+    def create_object(self, template:dict, node:str)->dict:
```

Comment:
I think the name of this method could be more descriptive. It sounds like its processing the template object and returning it? 

I'm also thinking that the template object would benefit from being its own dataclass.

Reply:
There's a template that gets referred to several times in this class, for example a base template gets created in `base_jsonld_template`. It might make sense for that template to be its own class `Template` and when it's created with nor parameters it could have the same values as the one created by `base_jsonld_template`